### ISSUE TITLE
How do you send audio to the app?

### ISSUE THREAD

**Issue by pilotkid** [tuckerbuchy/ledvis#5]
Hello, How do you send audio to the application is it just by playing or is there something special I need to do?

**Comment by tuckerbuchy** [comment]
Ummm, so in the README.md there is a link that will explain that for you. Are you running a linux distro?

**Comment by pilotkid** [comment]
Yes I am running raspbian but I am planning on trying ubuntu as soon as it reboots
